io.embrace.sdk/Editor/Data/AndroidNestedConfigurations.cs

```diff
@@ -41,6 +41,16 @@ public SdkConfig()
 
         public bool ShouldSerialize()
         {
+            // TODO: This is a workaround for a bug in the Embrace Android SDK
```

Comment:
is this a thing only in Unity? @jpmunz do we need to take care about it in RN?

Reply:
I had asked about that here: https://embrace-io.slack.com/archives/C03K25LSG3U/p1734019305072779?thread_ts=1733981554.735349&cid=C03K25LSG3U, if I'm understanding the issue correctly I think we were ok with it on RN